{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "toc_visible": true,
      "authorship_tag": "ABX9TyNJyhDd5QHc0dHrT/l32Zgq",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/leobioinf0/BIOSPERM/blob/main/Research_proposal_BIOSPERM_2022_pdf.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#1. PROJECT DATA\n",
        "\n",
        "TITLE OF THE PROJECT (ACRONYM): Biomarking the fish sperm epigenome and  molecular cargo for a sustainable aquaculture (BIOSPERM)\n",
        "\n",
        "#2. JUSTIFICATION AND NOVELTY OF THE PROPOSAL\n",
        "\n",
        "## 2.1. Adequacy of the proposal to the characteristics and purpose of the selected  modality  \n",
        "- Sperm deliver one half of the alleles and other regulatory factors that control the normal development  and vigor of cultured fish, yet the sustainability of such vital traits has largely been overlooked. The  main goal of the BIOSPERM project is therefore to develop novel paternal biomarkers of fertilization  success and embryo viability. For this, we will generate entirely new knowledge on the genetic and  epigenetic mechanisms occurring during the maturation of post-meiotic spermatozoa through an  ambitious holistic approach employing 'state-of-the-art'  'omics' technologies, and two piscine model  species, the marine teleost gilthead seabream (*Sparus aurata*) and the zebrafish (*Danio rerio*). To  determine how spermatozoa acquire sustainable traits during epigenomic reprogramming, we will also  investigate the role of novel endocrine signaling pathways recently discovered by the PI's group in fish  extratesticular ducts, as well as of extracellular seminal vesicles from these compartments. This  innovative research is designed to identify epigenetic marks and paternal molecules as candidate  biomarkers of sperm quality, which can provide valuable tools for the diagnosis of the fertility  competence and offspring performance in cultured fish. The results will have a new impact in selective  breeding programs, the optimization of methods for spawning induction, *in vitro* fertilization and sperm  cryopreservation, and in our understanding of how climate change (temperature, hypoxia and  acidification) will affect reproduction in the future. The project is thus **oriented** to contribute to the  resolution of specific problems linked to the potential intergenerational loss of epigenetic information  due to the continued use of assisted reproduction technologies and the effects of climate change, which  are critical challenges for future aquaculture sustainability. Such threats fall within the strategic action  in *Food, Bioeconomy, Natural Resources and Environment* from the [Spanish National Plan for Scientific  and Technical Research and Innovation (PEICTI) 2021-2023](https://www.ciencia.gob.es/InfoGeneralPortal/documento/e1f1deb1-7321-4dd9-b8ca-f97ece358d1c).\n",
        "\n",
        "## 2.2. Justification and expected contribution of the project to the generation of new  knowledge on the subject of the proposal.\n",
        "\n",
        "### **Starting hypothesis**\n",
        "- The control of the quality of the gametes in farmed marine fish remains a major challenge for the  sustainability of aquaculture. The obtention of high-quality gametes to assure the continuous production  of viable larvae is both critical for newly farmed species, as well as for other more domesticated species  under selective breeding, in which the monitoring and preservation of reproductive traits is  indispensable. However, the molecular basis of a good or bad egg or spermatozoon, as well as the mechanisms by which the culture conditions can affect the ability of gametes to fertilize and develop  into a normal embryo, are still poorly understood <sup>1</sup>.\n",
        "- For the male gametes, sperm motility is a primary determinant of fertility, and hence of sperm quality.  Accordingly, important advances in the knowledge of the ionic and osmotic factors controlling the  motility of the ejaculated spermatozoon have been accomplished <sup>2</sup>. However, the molecular mechanisms  controlling the acquisition of the motility competence during the formation of the spermatozoa, and how  external conditions can affect this process, are completely unknown. In addition, increasing evidence in  model fish species, such as the zebrafish, as in livestock animals, indicate that sperm-mediated  epigenetic inheritance is an important process that influences offspring development <sup>3-6</sup>. However, in  commercial fish these mechanisms remain largely unknown <sup>7</sup>, despite the fact that there is growing  concern about how the pre-existing epigenetic pattern in spermatozoa would be altered by assisted  reproductive technology practices (hormone therapies, cryopreservation), and what would be the risk of  transmission to the progeny. The lack of knowledge on the etiology of sperm motility disorders and on  the sperm components providing relevant paternal information in marine fish thus hampers the  establishment of biomarkers of gamete quality, and impedes a sustainable future aquaculture. Novel and  innovative research into these basic aspects of fish sperm function is therefore needed to provide viable solutions for the industry. The results of such research can lay the groundwork for the development of  useful paternal biomarkers to monitor and predict the sperm fertilization potential and proper epigenetic  marks in genetic selection efforts, artificial fertilization, and the handling and storage of gametes.\n",
        "\n",
        "### **Sperm maturation and the acquisition of motility competence**\n",
        "- After testicular spermatogenesis and spermiation, the fully differentiated spermatozoa are typically immotile and not capable of fertilizing the egg. They require a maturational phase, involving physiological but no morphological changes. This confers to the spermatozoon the ability to move, recognize and penetrate the egg. In mammals, sperm maturation takes place in the epididymis connecting the testis to the deferens duct, whereas in fish it occurs during storage and transit of spermatozoa through the extratesticular excurrent ducts (ETDs), i.e. efferent (ED) and sperm ducts (SD),that originate from the testis and end in the gonopore ([**Fig. 1A**]).\n",
        "- The epithelial cells lining the lumen of the fish ETDs are believed to be involved in the formation of  the seminal fluid, the nutrition of spermatozoa, and the elimination of necrotic cells, which maintain the  viability of the spermatozoon prior to ejaculation <sup>8</sup>. These cells also facilitate the creation of an ionic  gradient in the seminal fluid through the action of ion-dependent ATPases, that together with a slightly alkaline pH, seem to inhibit sperm motility <sup>2</sup>. Multiple aquaporins (selectively permeable to water, or to water and glycerol) expressed in a cell type- and region-specific manner along the ETD epithelium also play an  important role in the regulation of fluid  homeostasis across the ETD epithelial cells <sup>9</sup>. The regulation of the ion and water transport mechanisms in these cells is largely unknown, although some of them seem to be regulated by hormones, such as gonadotropins and progestins <sup>2</sup>. However, the changes in the  ionic and pH environment of the ETD fluid are not the major triggers for the acquisition of full sperm  motility in fish, since in the gilthead seabream, for instance, the ED (immature) spermatozoa activated  in seawater show much lower motility than the ejaculated (mature) spermatozoa ([**Fig. 1B**]). In addition, recent studies in salmonids show that the acquisition of full motility by the spermatozoa can occur in a  pH-independent manner <sup>10</sup>. These data therefore suggest the existence of yet unknown molecular  mechanisms in the ETDs that are required for the acquisition of sperm motility competence in fish.\n",
        "\n",
        "![Fig. 1](https://drive.google.com/uc?export=view&id=1foZLTjtHxDrAuStYVa-eUTVDerRcdgH5)\n",
        "- Fig. 1.:\n",
        "\t- (**A**) Anatomy of the seabream testis and extratesticular  ducts (ETDs).\n",
        "\t- (**B**) Percentage of motility of seabream spermatozoa from the efferent duct (SPZ<sub>ED</sub>) or ejaculated (SPZ<sub>EJ</sub>).  \\*\\*\\*, statistically different (P < 0.001). CerdÃ  and col. (unpublished).\n",
        "\n",
        "### **The sperm epigenome and molecular cargo and its transmission to the offspring**\n",
        "- During spermatozoa differentiation, the chromatin DNA is methylated and compacted, which is accompanied by changes in the structure of nuclear proteins. DNA methylation occurs at cytosine  residues of the CpG dinucleotides, and is considered a powerful mark to silence genes. In some fish,  such as the seabream and zebrafish, the DNA is exclusively condensed with histones, while in mammals  and other fish species DNA compaction follows the partial or complete displacement of histones from  the DNA-packing nucleosomes by other sperm nuclear basic proteins, such as protamines or protamine like proteins <sup>11</sup>. As a result of such tight nuclear compaction, transcription is believed to be repressed in  the spermatozoon, although mitochondrial translation may remain active <sup>12</sup><sup>,</sup><sup>13</sup>. In addition, a variety of  post-translational modifications (PTMs) of histones mainly in the form of methylation or acetylation of  amino acids in the N-terminus of these proteins also occur at specific genomic sites <sup>14</sup>, which correlate  with either transcriptionally active or silenced regions of the genome. Thus, both DNA methylation and  histone PTMs confer epigenetic marks to the sperm that influence chromosome architecture and gene  expression.\n",
        "- The epigenetic information carried by mammalian and fish sperm is transmitted to the offspring,  producing changes that may affect the further performance of adult animals. For instance, it has been  shown that the rearing environment induces specific changes to salmon (*Salmo salar*) sperm DNA  methylation which is inherited by the offspring, while sperm-mediated paternal effects on phenotype  have been demonstrated in zebrafish in response to increased sperm competition and paternal social  status, stress conditions, or exposure to pollutants <sup>4</sup><sup>,</sup><sup>6</sup><sup>,</sup><sup>7</sup>. However, in commercial fish information on how  the environmental or handling conditions may affect the sperm epigenetic marks leading to male  infertility or developmental defects is still very scarce <sup>7</sup>. In addition, it is not known whether changes in  the fish sperm epigenome only occur during the differentiation of spermatozoa in the testis, or as in the  mammalian epididymis <sup>15</sup>, epigenetic reprogramming may also take place in the fish ETDs.\n",
        "- The mature sperm also carry protein coding RNAs (mRNAs), as well as different classes of non coding RNAs (ncRNAs), that together with epigenetic marks are transmitted to the progeny. The  ncRNAs include long ncRNAs (lncRNAs) and small ncRNAs, such as microRNAs (miRNAs) and Piwi interacting RNAs (piRNAs) among others, which can regulate gene expression and chromatin structure,  and hence, epigenetic mechanisms <sup>3</sup><sup>,</sup><sup>16</sup>. These RNAs are thought to be the remnants of the  spermatogenesis process, or in the case of mammals, they may be delivered to the maturing spermatozoa  via epididymosomes, extracellular vesicles (exosomes) released by epididymal epithelial cells that are  internalized by the spermatozoon <sup>3</sup>. In mammals, sperm-borne mRNAs and miRNAs have been  associated with the sperm cell fertility status and early embryo development, and therefore some of these  RNAs have been suggested as potential biomarkers in ejaculated spermatozoa to predict male fertility  potential <sup>3</sup>. In teleosts, similar information is still limited, although recent studies in zebrafish show that  stress can alter the levels of miRNAs and piRNAs in the sperm that could be transmitted to the  offspring <sup>4</sup>, or that low quality sperm can induce low fertilization and high malformation rates in  embryos, together with an altered embryonic miRNA profile <sup>17</sup>. It is therefore possible that paternal  miRNA and/or other ncRNAs delivered in spermatozoa could also influence developmental outcomes  in commercial fish. In addition, recent studies have isolated fish seminal exosomes, which miRNA and  piRNA composition changes between normal males and sex-reversed females <sup>18</sup>. These findings thus  suggest that exosomes released by the ETD epithelial cells may be important to establish the RNA cargo  of mature fish spermatozoa and determine their fertilization and post-fertilization potential.\n",
        "\n",
        "### 2.2.1. Starting hypothesis\n",
        "- During a previously financed project (AGL2016-76802-R, SPERMIOPORIN), the PI's group carried  out a transcriptomics study during the differentiation and maturation of seabream spermatozoa that  revealed the enrichment of a high number of transcripts potentially involved in transcription, translation  and chromatin organization, as well as the upregulation of the gonadotropin-releasing hormone receptor  (GnRHR) and platelet-derived growth factor (PDGF) signaling pathways <sup>19</sup>. Recent experimental  investigation showed the expression of these endocrine pathways in sequential segments of the ETDs,  where their paracrine signaling stimulates the motility of spermatozoa in a transcription-dependent  manner ([Fig. 2A,C]). Notably, the same observations were made in the zebrafish, which as in the  seabream the spermatozoa chromatin lack protamines ([Fig. 2B,D]). Further, we found that both GnRH  and PDGF induce *de novo* transcription of motility-related genes in seabream immature spermatozoa in  vitro, suggesting that the acquisition of full motility may be transcriptionally regulated by endocrine  signalsfrom the ETD. However, these pilot studies did not rule out potential post-transcriptional changes  in the spermatozoa steady state RNA levels, and therefore direct evidence of this mechanism is lacking. Nevertheless, *in vitro* incubations experiments with labelled amino acids show that zebrafish and  seabream immature spermatozoa are translationally active (Castro-Arnau et al., in preparation).\n",
        "\n",
        "![Fig. 2](https://drive.google.com/uc?export=view&id=1MmEZeljTpnudQeSet_wkcX3B9gihuk2K)\n",
        "- Fig. 2.: A novel endocrine signaling pathway in seabream and zebrafish ED.\n",
        "\t- (**A**-**B**) Localization of GnRH production and expression of PDGF-encoding transcripts in ED epithelial cells (arrows).\n",
        "\t- (**C**-**D**) Stimulation of motility of ED spermatozoa by treatment with GnRH and PDGF *in vitro*, which is blocked by the transcription inhibitor Actinomycin D (AcD). ***, P < 0.001. CerdÃ  and col. (unpublished)\n",
        "\n",
        "- Interestingly, in the zebrafish we also found that ED sperm-mediated *in vitro* fertilization results in  lower percentage of fertilized embryos and hatching rates, together with a relatively higher number of  embryos with abnormal tail development (up to ~30%) at 48 h post-fertilization, with respect to those  obtained with ejaculated sperm ([Fig. 3A,B]). However, these  developmental defects  could be partially  rescued by *in vitro* treatment of immature  sperm with GnRH or  PDGF for ~16 h prior to  fertilization ([Fig. 3C]). These observations  suggest the existence of  soma to germ cell  signaling pathways  during sperm maturation  in fish that regulate not  only the acquisition of  full motility, but also  induce changes in  spermatozoa that are  transmitted to the offspring and influence normal development. Our data may also imply that post meiotic spermatozoa might not remain completely transcriptionally silent, but that these mechanisms may play an important role for male fertility potential. Preliminary experiments also show that the rescue  of the embryo malformations by GnRH treatment of immature sperm is blocked by both transcription  and translation inhibitors, whereas the inhibitors do not affect the positive effect of PDGF. These data  further suggest that the mechanisms triggered by GnRH in spermatozoa and transmitted to the embryo  may be related to the inheritance of transcribed RNAs and/or translated proteins, while that of PDGF is  possibly based on the transmission of hormone-induced changes in the spermatozoon epigenetic marks.\n",
        "\n",
        "![Fig. 3](https://drive.google.com/uc?export=view&id=1WXa4HvEkhnqeP5gj_cjRgSVcQOEZL38y)\n",
        "- Fig. 3.: Hormones induce sperm-borne factors required for normal embryo  development in zebrafish.\n",
        "\t- (**A**) % of fertilization, hatching and of embryos showing  abnormal tail development at 48 h post-fertilization (**B**) from eggs fertilized with  immature (SPZ<sub>ED</sub>) or ejaculated (SPZ<sub>EJ</sub>) sperm.\n",
        "\t- (**C**) Rescue of the SPZ<sub>ED</sub>-triggered  embryonic defects by treatment with GnRH or PDGF prior to fertilization. \\*, P < 0.05; \\*\\*,  P < 0.01; \\*\\*\\*, P < 0.001. CerdÃ  and col. (unpublished).\n",
        "\n",
        "- Based on our pilot data described above, **we hypothesize that a second reprogramming wave of  the fish post-meiotic spermatozoon epigenome, together with an activation of transcriptional  mechanisms, occur during the maturation phase in the ETDs, which determines the spermatozoon  motility competence and further offspring performance**. We also postulate that some of these  mechanisms are under direct endocrine regulation, while seminal exosomes released by ETD epithelial cells may also contribute to the sperm-borne RNA cargo. We finally propose that the elucidation of  these molecular events will generate innovative information to establish new biomarkers of sperm  quality in marine fish of high interest for a sustainable aquaculture.\n",
        "\n",
        "## 2.3. Justification and expected contribution of the project to solve specific problems  linked to the selected thematic priority\n",
        "- The BIOSPERM project is oriented to contribute to the resolution of specific problems related to the thematic priority 6 of the PEICTI 2021-2023, Food, Bioeconomy, Natural Resources and Environment.  One of the challenges in this strategic line is to provide solutions for a sustainable aquaculture through  innovation actions to produce healthy and quality food. As highlighted by the European Aquaculture  Technology and Innovation Platform (EATiP) (2017 Vision; [EATiP](https://eatip.eu/)), for marine fish  aquaculture this objective involves the development of selective breeding to target important traits, such  as adaptation to alternative feed sources, disease resistance, feed efficiency, fillet yield, flesh quality, or  nutritional profile. The BIOSPERM project aims to contribute to solve specific challenges facing these  breeding programs, such as broodstock selection, optimization of mating through artificial fertilization  to control inbreeding, or the conservation (cryopreservation) of viable sperm, by the obtention of  paternal molecular biomarkers. These markers will provide valuable tools to monitor the quality of  sperm for the sustainable production of 'robust' animals, resistant to disease, stress and changing  environmental conditions.\n",
        "\n",
        "#3. OBJECTIVES, METHODOLOGY AND WORK PLAN\n",
        "\n",
        "## 3.1. General and specific objectives\n",
        "- The main objective of BIOSPERM is the **identification of epigenetic marks and paternal molecules  as candidate biomarkers of sperm quality to evaluate the fertilization success and offspring  viability in cultured fish**. For this, we will carry out for the first time in a marine teleost a holistic  approach to identify genetic and epigenetic changes occurring in post-meiotic spermatozoa during their maturation phase in the ETDs. In addition, we will investigate the role of the newly discovered endocrine  signaling pathways between the ETD epithelial cells and sperm, as well as of ETD exosomes, in the  configuration of the epigenetic signature and molecular cargo of the mature spermatozoon. The project  is divided into the following specific objectives:\n",
        "\n",
        "### Objective 1: Investigation of chromatin reprogramming processes during the maturation of seabream post-meiotic spermatozoa.\n",
        "- **Rationale:** Our preliminary data suggest that GnRH and PDGF might trigger transcription in the  immature spermatozoa to promote the acquisition of full motility ([Fig. 2](To=A9i-tkQ4liBl)). In addition, these endocrine  signals seem to activate epigenetic mechanisms that are inherited by the offspring ([Fig. 3]). In order to obtain experimental support for this hypothesis, the initial task of the project will be focused on  obtaining a comprehensive view of the changes in the seabream sperm genome and epigenome during  the maturation phase. To accomplish this, we will employ three complementary genome-wide approaches on immature and mature spermatozoa:\n",
        "\t- (i) whole-genome methylome analysis to gather a  map of potentially repressed and activated genomic regions;\n",
        "\t- (ii) liquid chromatography coupled to mass  spectrometry (LC-MS) analysis of histone PTMs to obtain additional epigenetic markers; and\n",
        "\t- (iii) high throughput chromosome conformation capture (Hi-C) to measure three-dimensional (3D) proximity of  genomic loci and reveal interactive regions of the chromatin, which are potentially involved in the  regulation of gene expression.\n",
        "- The combined dataset will uncover the sperm nuclei reprogramming mechanisms occurring during the maturation of spermatozoa, and will set the basis to investigate gene  transcription and epigenetic mechanisms occurring in immature spermatozoa in vivo and in response to  GnRH and PDGF *in vitro* in objectives 2 and 3.  \n",
        "\n",
        "### Objective 2: Profiling of the molecular cargo in the seabream immature and mature spermatozoon  and investigation into its cellular origin.\n",
        "- **Rationale:** In order to identify paternal RNAs that may be important for sperm function and early  embryo development, as well as to confirm whether transcription regulatory mechanisms are activated  in developing spermatozoa as inferred from the results of the objective 1, here we will determine changes  in the RNA cargo during the maturation of seabream spermatozoa. As a first step, we will use transcriptome RNA sequencing (RNA-seq) of ETD and ejaculated spermatozoa, focusing on mRNAs,  lncRNAs and miRNAs. The sperm nuclear origin of these molecules will be further investigated by  chromatin immunoprecipitation sequencing (ChIP-seq) using antibodies against modified histones associated with transcription activation/repression based on the information from objective 1 and the  sperm RNA-seq data. To investigate the fraction of the sperm molecular cargo with an extracellular  origin, we will isolate exosomes from the ETD seminal fluid and characterize their RNA and protein  content by RNA-seq and proteomics, respectively.\n",
        "\n",
        "### Objective 3: Investigation of the GnRH and PDGF regulated transcription and epigenetic  mechanisms in immature fish spermatozoa.\n",
        "- **Rationale:** Although our pilot data indicate that GnRH and PDGF can promote full sperm motility and induce changes in the spermatozoon inherited by embryos, conclusive evidence for the presence of these endocrine signal transduction pathways in immature spermatozoa is still laking. In addition, hormone induced *de novo* gene transcription in ETD spermatozoa for the acquisition of full motility and normal embryo development, as well as the presumed PDGF triggered changes in the epigenome, remain to be  demonstrated. To address these questions, we will first characterize the GnRH and PDGF signaling  pathways in seabream and zebrafish immature spermatozoa, and investigate hormone-induced *de novo* gene transcription by using bromouridine (BrU) sequencing (Bru-seq). To identify the RNAs regulated  by GnRH and the PDGF-induced epigenetic mechanisms in the spermatozoa that are transmitted to the  embryo, we will investigate the sperm factors underlying the abnormal larval phenotype arising from  zebrafish embryos fertilized with immature sperm. For this, we will employ RNA-seq of abnormal  embryos, morpholino-mediated gene knockdown in viable embryos, and targeted methylation analysis  on both sperm and embryos based on the data from objectives 1 and 2.\n",
        "- The flow-chart of the objectives is illustrated below.\n",
        "\n",
        "![flow-chart](https://drive.google.com/uc?export=view&id=1DoDWkUmiMpWSFU9A4mb3k1j_a8fGWisE)\n",
        "\n",
        "- The objectives of the BIOSPERM proposal represent a follow-up of the previous PI's project  SPERMIOPORIN and are based on novel findings that provide a reasonable foundation for the proposed  hypothesis, as well as for the **feasibility** of the goals set out in the present project. The objectives are  **highly innovative** because they aim to generate new knowledge for the development of novel paternal biomarkers of fertilization success and offspring viability in marine fish, based on the reprogramming  of the spermatozoon epigenome and molecular cargo during maturation, which are key to the control of  high-quality gametes in the aquaculture of the gilthead seabream and other marine teleosts.\n",
        "\n",
        "## 3.2. Description of the methodology  \n",
        "\n",
        "### Objective 1: Changes in the chromatin state during seabream sperm maturation\n",
        "\n",
        "#### Task 1.1. Genome-wide sperm DNA methylation analysis.\n",
        "- Samples of immature (ETD) and  ejaculated spermatozoa will be collected from reproductively active males (2-3 years old) by  fluorescence-activated cell sorting (FACS), as previously published by the PI's group <sup>19</sup>, in order to avoid contamination by other cell types and exosomes that may be present in the seminal fluid. The genome wide DNA methylation pattern analysis of immature and mature spermatozoa will be carried out by  Enzymatic Methylation-seq (EM-seq), a more accurate and reliable system than whole genome bisulfite  sequencing, which minimizes DNA damages and produces high quality libraries <sup>20</sup>. For this, the DNA  from spermatozoa (n = 5 fish per group, 10 samples in total) will be extracted using appropriate  commercial kits and fragmented into ~200 bp. The EM-seq libraries will be prepared from sheared DNA  using an enzymatic methyl-seq kit (New England BioLabs), and sequenced on a NovaSeq 6000  sequencer (Illumina) with a paired-end mode with a read length of 2x150 bp reads (>30Gb/sample, with  a x30 coverage) at the National Center of Genomic Analysis (CNAG, Barcelona). Adapter trimmed  reads will be mapped to the seabream genome (GCA_900880675.1), which the PI's group has recently  reannotated <sup>19</sup>, by using the BSMAP software. Differentially methylated regions (DMRs) between  immature and mature spermatozoa will be identified using different softwares (Bismark, MethylKit,  DMRichR) as previously described <sup>21</sup>. The DMR-containing genes will be annotated and related signaling pathway analyzed using the PANTHER software as previously published by the PI's group <sup>19</sup>.\n",
        "\n",
        "#### Task 1.2. Proteomic analysis of histone PTMs.\n",
        "- The histone PTMs present in ETD and ejaculated spermatozoa will be analyzed by LC-MS, which will allow the identification and quantification of the  histone modifications <sup>22</sup>, at the Protein Analysis Unit of the Biomedical Center (University of Munich,  LMU, Germany) (see Section [3.3.1]). Initially, the PTMs to be analyzed will be: H3K4 unmodified,  mono, di, trimethylated; H3K9K14 unmodified, mono, di, trimethylated, mono and diacetylated;  H3K18K23 unmodified, mono and diacetylated; H3K27K36 unmodified, mono, di, and trimethylated,  mono and diacetylated; H4K5K8K12K16 unmodified, mono, di, tri and tetraacetylated; and H4K20  unmodified, mono, di, trimethylated. We will apply the bottom-up strategy since it is highly sensitive  and high-throughput <sup>22</sup>. The histones from spermatozoa isolated as in Task [1.1.] (n = 5 fish per duplicate,  20 samples in total) will be acid extracted, chemically acylated to prevent short hydrophilic peptides  and to maintain certain PTMs connectivity, further digested with trypsin, and the resulting peptides  desalted prior to LC-MS analysis. The histone peptides generated will be analyzed using two  complementary MS scan modes: a combined data dependent acquisition with parallel reaction  monitoring (DDA-PRM) <sup>23</sup>, and a variant of the PRM mode, the so-called Internal Standard triggered Parallel Reaction Monitoring or IS-PRM <sup>24</sup>, available in Orbitrap Exploris-480 instruments (Thermo  Fisher Scientific). Both approaches will benefit from the presence of isotopically-labelled peptide  standards (JPT Peptide Technologies), which will ensure the identification of the correct peptide signal,  the normalization across measurements and the triggering of the PRM scan mode.\n",
        "\n",
        "#### Task 1.3. Hi-C sequencing.\n",
        "- In order to analyze the differences of genome organization during the  maturation of spermatozoa, Hi-C libraries will be generated from FACS collected ETD and ejaculated  spermatozoa (n = 3 fish per group). Cells will be fixed with 1% formaldehyde and cross-linked  chromatin will be digested with convenient restriction enzymes following the methods described by  Vara et al. <sup>25</sup>. After end repair using biotinylated dNTP, spatially closed DNA fragments will be ligated.  Following mechanical fragmentation, biotinylated fragments will be purified to generate sequencing Hi C-sequencing libraries (6 in total), which will be sequenced in a paired-end mode with a read length of  2x150 bp on NovaSeq 6000 S4 (Illumina) using TruSeq SBS Kit v4-HS (Illumina) at the CNAG. After  mapping to the seabream genome and filtering, Hi-C valid pairs will be used to generate contact matrices  at various resolutions. In collaboration with Dr. F. Le Dily from the Center of Genomic Regulation  (CRG, Barcelona), these datasets will be analyzed at different levels of organization (chromatin  compartmentalization, topologically associated domains, and if possible also loops) in correlation with  other high-throughput data generated in Tasks [2.1.] and [2.2.] (RNA-Seq, ChiP-Seq) using available  bioinformatic tools, such as TADbit and Hi-C explorer.\n",
        "\n",
        "### Objective 2: Composition and origin of the molecular cargo in immature and mature  seabream spermatozoa\n",
        "\n",
        "#### Task 2.1. Transcriptome profiling of immature and mature seabream spermatozoa.\n",
        "- Long and  small RNAs will be extracted from immature and mature spermatozoa collected as above (n = 4 separate  pools, each replicate being a pool of cells collected from three different males) employing commercial  kits. The RNAs will be used to construct 16 unstranded low input RNA-seq libraries (8 for long RNAs  and 8 for small RNAs). For long RNAs, the libraries will be constructed and subsequently sequenced at  CNAG as recently carried out by the PI's group <sup>19</sup>, whereas for small RNAs the libraries will be  constructed and sequenced (Illumina NextSeq 2000; 10 M reads, 1x50 bp) at the CRG sequencing service. Long RNA-seq reads will be mapped and annotated using the improved version of the seabream reference genome with STAR v2.5.3a using ENCODE parameters for long RNAs, as we have recently  done <sup>19</sup>. Differential expression analysis will be performed with the DESeq R statistical package to  determine which RNAs are differentially expressed during sperm maturation, and gene ontology  pathway analysis will be carried out using the KEGG, PANTHER and DAVID databases as previously  described <sup>19</sup>. Differentially expressed miRNAs will also be mapped against the seabream genome, and  annotated against teleost miRNAs in the [miRbase], the [Gasterosteus_aculeatus.BROADS1.ncrna](https://www.ensembl.org/Gasterosteus_aculeatus) and the [FishmiRNA](https://fishmirna.org/) databases. The putative miRNAs targets will be identified using different softwares (RNAhybrid, miRanda, Targetscan) using the open-reading frames of the assembled seabream  transcriptome as previously described <sup>26</sup>. Other differentially expressed sncRNAs (e.g. piRNAs) will also  be identified and annotated as much as possible using the [RNAcentral database](https://rnacentral.org/). Finally, the differential expression of transcripts will be confirmed by real-time qPCR.\n",
        "\n",
        "#### Task 2.2. ChIP-seq during spermatozoa maturation.\n",
        "- To identify regions of the genome under  transcriptional regulation during the maturation of spermatozoa we will carry out comparative ChIP-seq  analysis between ED and ejaculated spermatozoa using two different antibodies for histone  modifications indicative of 'active/inactive' genes (selected from Task [1.2.]). In collaboration with Dr.  A. Ruiz-Herrera from the Institute of Biotechnology and Biomedicine (IBB-UAB), the chromatin of  FACS-sorted immature and mature spermatozoa (n = 3 fish per group) will be cross-linked, extracted  and fractionated into ~200 bp DNA fragments as previously described <sup>25</sup>. An aliquot of each sample will  be used as input and the remaining will be immunoprecipitated. Subsequently, the crosslink of the DNA  will be reversed, and the fragment ends repaired and ligated to the adaptors for library construction. A  total of 18 Illumina Truseq ChiP libraries (Input, antibody 1 and antibody 2 for each fish) will be  constructed and sequenced (Novaseq6000 150PE, 2x150 bp) 18 Gb/sample) at the commercial company Macrogen. The sequencing reads will be trimmed and mapped to the seabream reference genome, and  the coverage distribution across the genome will be determined. The identification and visualization of  areas in the genome that have been enriched with aligned reads (peak calling) will be performed using  the Model-based Analysis of ChIP-seq (MACS2) algorithm and the IGVtools software, respectively <sup>25</sup>. The ChIP data, together with the information from Tasks [1.2.], [1.3.] and [2.1.], will be employed to localize  'active' chromatin regions in the spermatozoon nucleus using fluorescence in situ hybridization (FISH) for selected loci and immunofluorescence microscopy (IF) for histones.\n",
        "\n",
        "#### Task 2.3. Isolation and molecular characterization of seminal fluid exosomes in the ETD.\n",
        "- Following procedures recently established in the PI's group in collaboration with Dr. I. Pividori (IBB UAB) ([**Fig. 4**]), exosomes will be purified from the seminal fluid of the seabream ED by sequential  ultracentrifugation. The exosome RNA composition (long and small RNAs) from isolated exosomes (n = 3  fish per triplicate) will be analyzed by  shotgun sequencing (9 RNA-seq  libraries for long RNA, and 9 libraries  for small RNA). The RNA-seq reads  will be mapped and annotated, as  described in Task [2.1.] The proteomic  profile of exosomes (n = 3 fish per  triplicate) will be analyzed by label free LC-MS/MS in collaboration with  Dr. M. Carrascal from the Proteomics  Laboratory LP/CSIC-UAB. The  RNA-seq and proteomics results will  be confirmed, respectively, by qPCR  or western blot on isolated exosomes.  For the latter, we will use commercial  antibodies showing cross-reaction  with the seabream proteins, or will  produce seabream specific polyclonal antibodies against proteins of interest (as the PI's group has done in previous projects).\n",
        "![Fig. 4](https://drive.google.com/uc?export=view&id=1iTfTyZnFC-kjX7ZgJuy5QmFlN_6OXgKK)\n",
        "- Fig. 4.: Purification of  seabream ED seminal exosomes.\n",
        "\t- (**A**) Transmission  electron microscopy  images of purified  exosomes.\n",
        "\t- (**B**) Particle  size distribution and  concentration of exosomes analyzed  using nanoparticle  tracking analysis performed using NanoSight NS300. B inset, detection of heat  shock protein 90 in  protein extracts from  exosomes by immunoblotting. CerdÃ , Pividori and  col. (unpublished).\n",
        "\n",
        "### Objective 3: Molecular mechanisms regulated by GnRH and PDGF signaling pathways  in immature spermatozoa\n",
        "\n",
        "#### Task 3.1. Pharmacological characterization of the GnRHR and PDGF signaling pathways in  immature spermatozoa.\n",
        "- The initial experiments will test the effect of inhibitors of different  components of the GnRHR and PDGF signaling pathways on the hormone-induced motility of the ETD  spermatozoa *in vitro* in both seabream and zebrafish. The inhibitors will include commercially available specific blockers of the GnRH and PDGF cognate receptors, protein kinase A, or mitogen-activated  protein kinase kinase, as well as of transcription factors, such as cAMP-responsive element binding  protein, signal transducer and activator of transcription or the transcription factor c-Jun. The seabream  GnRH (sbGnRH) and recombinant PDGF-BB will be purchased from Bachem and ThermoFisher  Scientific, respectively. The immature spermatozoa will be incubated *in vitro* with the hormone vehicles  (controls), sbGnRH or PDGF-BB, in the presence or absence of the inhibitors, for approximately 16 h  at 18ÂºC, activated in seawater, and the sperm motion kinetics subsequently determined by computer assisted sperm analysis (CASA) available at the PI's group <sup>19</sup>. Further determination of the Ca<sup>2+</sup> and  cAMP levels in immature spermatozoa after hormone treatment, as well as immunoblotting using  commercial antibodies for phosphorylated and non-phosphorylated forms of selected kinases and  transcription factors, will confirm the intracellular activation of each pathway in the spermatozoon.\n",
        "\n",
        "#### Task 3.2. Determination of hormone-induced activation of transcription and/or translation in  ETD spermatozoa.\n",
        "- Using seabream and zebrafish, we will investigate whether transcription and/or  translation is hormonally activated in ETD spermatozoa. For this, we will carry out time-course *in vitro* experiments (up to 6 h, based on preliminary trials conducted by the PI's group) of incorporation of  fluorescently labelled lysine residues in ETD spermatozoa incubated in the presence or absence of  sbGnRH and PDGF-BB, and with or without inhibitors of transcription (actinomycin D) and cytosolic  and mitochondrial translation (cycloheximide and chloramphenicol, respectively). For these experiments, we will employ the FluoroTectâ¢ GreenLys *in vitro* Translation Labeling System kit  (Promega), followed by immunoblot of translation products using an anti-BODIPYÂ®-FL antibody  (Invitrogen) and an anti-prohibitin antibody for data normalization.\n",
        "- To confirm hormone-activated *de novo* transcription in immature spermatozoa, we will employ Bru seq, which is based on the metabolic pulse-chase labeling of nascent RNA with BrU, following the  methods described by Paulsen et al. <sup>27</sup>. Seabream and zebrafish ETD spermatozoa (n = 3 replicate fish) will be treated with vehicle (controls), sbGnRH or PDGF-BB *in vitro* (as above) in the presence of BrU for up to 6 h at 18ÂºC, the RNA extracted, and subsequently immunoprecipitated using a mouse anti BrdU antibody (BD Pharmingen). The labelled RNAs will be then eluted and used to construct 9 long  RNA-seq libraries for each species (18 libraries in total), which will be sequenced, annotated and  analyzed as described in Task [2.1.] (in this case, using also the zebrafish reference genome).\n",
        "\n",
        "#### Task 3.3. Identification of endocrine-activated molecular mechanisms in ETD spermatozoa  influencing flagellar motility and embryo development.\n",
        "- To investigate the transcriptional and epigenetic mechanisms activated by GnRH and PDGF in ETD spermatozoa for the acquisition of full  motility and normal offspring development, we will employ seabream and zebrafish sperm, as well as zebrafish embryos. The specific approaches that will be taken are described below  ([**Fig. 5**]):\n",
        "\t1. To investigate GnRH and PDGF regulated sperm motility, the expression level of mRNAs  identified by Bru-seq analysis with predicted functions in sperm motility (e.g. aquaporins, ion channels, flagellar proteins) will be analyzed by qPCR. The results will be confirmed by immunoblotting using commercial antibodies that cross-react with the fish proteins and seabream specific antibodies available  at the PI's group. Further functional experiments using inhibitors (e.g. ion channels blockers or  aquaporin antibodies) and CASA will establish the physiological significance of the factors identified.\n",
        "\t2. To confirm that the GnRH and PDGF triggered mechanisms in spermatozoa that are transmitted  to the offspring are different, we will determine again whether the rescue of the abnormal larval  phenotype in zebrafish mediated by GnRH and PDGF treated immature sperm (Fig. 3C) is blocked by  transcription and translation inhibitors. For this, immature sperm will be exposed to GnRH or PDGF, in  the presence or absence of actinomycin D, cycloheximide or chloramphenicol, before fertilization, and  the phenotype of embryos fertilized with the two types of sperm will be analyzed as previously carried  out by the PI's group.\n",
        "\t3. To determine the direct transmission of sperm RNAs into early embryos, we will first test by  qPCR the level of expression of long RNAs (mRNAs and lncRNAs) identified from Bru-seq in 2-4 cell embryos (before activation of the zygotic genome) fertilized with immature or GnRH-treated sperm. In  parallel, we will perform RNA-seq of long and small RNAs on the two types of embryos, as described  in Task [2.1.], to identify additional RNAs not detected in the Bru-seq experiments as candidate inherited  factors. Since the percentage of abnormal larvae arising from embryos fertilized with immature sperm  vary from ~10 to 30%, for these experiments the batches of embryos to be analyzed will be monitored  up to 48 h post-fertilization to ensure that they result in the highest percentage of abnormal larvae.\n",
        "\t4. To identify PDGF-regulated DNA methylation marks in sperm inherited by embryos, the changes  in the methylation pattern of specific genomic loci in sperm treated or not with PDGF, in 1-2 cell  embryos fertilized with untreated or PDGF-treated immature sperm, and in 48 h larvae arising from embryos fertilized with immature or mature sperm, will be investigated by pyrosequencing targeted  methylation analysis using EZ DNA Methylation-Gold Kits (Zymo Research) and a PyroMark Q96  pyrosequencer (Qiagen), at the Fish Physiology and Genomics lab (LPGP, INRAE, France) (see Section  [3.3.1]), as previously described <sup>28</sup>. All embryos/larvae will be analyzed individually, which together the information from Task [1.1.], will allow to distinguish the methylation marks specific of the paternal  alleles. The primers will be designed using MethPrimer (https://www.urogene.org/methprimer) based  on the data from objectives 1 and 2 and the embryo RNA-seq (point 3 above).\n",
        "\t5. Finally, functional confirmation of the role of paternal RNAs in early embryogenesis will be  carried out by using gene knockdown methods based on the microinjection of morpholinos into 1-2 cell embryos fertilized with mature sperm, and further phenotype analyses, as previously described <sup>29</sup>.\n",
        "\n",
        "![Fig. 5](https://drive.google.com/uc?export=view&id=1j3t2fCwVOTtcZyVPxBkwAF8eQyaKaAj6)\n",
        "- Fig. 5.: **Flow-chart of the different approaches that will be carried out in Task 3.3.**. The numbers indicated in the scheme refer to the different methodologies explained in the text.\n",
        "\n",
        "## 3.3. Working plan and chronogram of activities\n",
        "### 3.3.1. Research and working teams\n",
        "\n",
        "1. The **Research team** is composed by:\n",
        "    1. [Dr. Joan CerdÃ ](https://orcid.org/0000-0003-2568-6398) (JC), PI of the present proposal, who is IRTA Researcher A and the leader of the  Comparative Molecular Physiology (CMP) group at the Institute of Biotechnology and Biomedicine  (IBB) from the Universitat AutÃ²noma de Barcelona (UAB). He is expert in fish reproductive physiology,  with >30 years of experience, and his research has received continuous funding from the Spanish  Ministry of Science and Innovation since 2001 (7 consecutive individual projects). During the last years,  his work has been focused on the molecular physiology of seabream spermatozoa.\n",
        "    2. [Dr. Roderick N. Finn](https://orcid.org/0000-0002-8776-3945) (RNF), Research Professor from the University of Bergen (Norway) associated to IRTA since 2018 until 2029 (agreement available upon request), and integrated in the CMP  group at the IBB-UAB. He is expert in computational phylogenetics and is also experienced in fish  proteomics. He has been closely working with the PI for >10 years.\n",
        "2. The **Working team** is composed by six members:\n",
        "    1. [Angel Rey](https://ibb.uab.cat/wp-content/themes/viral/modules/ibb_membres/view_membre.php?CodiMembre=768) (AR), PhD researcher from the CMP group with experience in RNA-seq and shotgun  proteomics, microinjection of zebrafish embryos, and RNA interference (RNAi).\n",
        "    2. [Dr. Ignasi FornÃ©](https://orcid.org/0000-0003-0309-907X), ([IF](https://www.researchgate.net/profile/Ignasi-Forne)), Technical Head of the Proteomics Facility from  LMU (Germany), expert in the analysis of proteins using MS in a wide variety of biological systems,  and particularly on histone PTMs <sup>23</sup><sup>,</sup><sup>30-32</sup>, including fish <sup>33</sup><sup>,</sup><sup>34</sup>\n",
        "    3. [Oscar Conchillo-SolÃ©](https://orcid.org/0000-0003-4266-246X) (OC), bioinformatics technician at the IBB-UAB and member of the Computational Biology group. He is UNIX/Linux  systems and data center manager at the IBB\n",
        "    4. [JosÃ© Vicente Roig](https://orcid.org/0000-0001-8288-859X) (JVR), Biotechnologist from the  Spanish company Rara Avis Biotec S.L.(RAB) (https://www.raraavis-bio.com/, see also Section [4.5.]),  who is highly skilled in omics technologies and recently started his PhD thesis on the molecular  characterization of fish gametes produced by hormone induction (co-advised by the PI);\n",
        "    5. [Dr. Catherine LabbÃ©](https://orcid.org/0000-0002-6058-9684) (CL), Research Director at the LPGP-INRAE expert in biotechnologies for genome preservation  in aquatic species, particularly in fish.\n",
        "    6. [Dr. Audrey Laurent](https://orcid.org/0000-0002-7643-6017) (AL), Principal investigator also from  LPGP-INRAE expert in fish spermatozoon epigenetics.\n",
        "\n",
        "- Five of the project members are **experts in genomic, methylomic, transcriptomic or proteomic analyses** in order to manage the considerable workload in NGS data processing and bioinformatics  involved in the project.\n",
        "\n",
        "3. In addition, as indicated in Section [3.2.], the PI will collaborate with other external researchers (not  officially included in the proposal), who are experts in different techniques employed in the project and  that have expressed their interest to cooperate:\n",
        "    1. [Dr. FranÃ§ois Le Dily](https://orcid.org/0000-0002-8324-7927), Researcher at the Center of Genomic Regulation (CRG) in Barcelona and expert in high throughput chromosome conformation capture related methods, in particular on Hi-C in germ cells. He  will assist in Task [1.3.]\n",
        "    2. [Dr. Aurora Ruiz-Herrera](https://orcid.org/0000-0003-3868-6151), Professor at the  Department of Cellular Biology, Physiology and Immunology from UAB and IBB group leader. She is expert in comparative genomics, mammalian evolution and chromosomal instability, and is highly  skilled in ChIP and Hi-C methods on germ cells. She will assist in Tasks 1.3 and [2.2.]\n",
        "    3. [Dr. Isabel Pividori](https://orcid.org/0000-0002-5266-7873), Professor in the Chemistry Department of UAB and IBB group  leader and expert in exosomes as biomarkers for breast cancer. She will help in Task [2.3.], following the  results already obtained together with the PI.\n",
        "    4. [Dr. Montserrat Carrascal](https://orcid.org/0000-0002-0205-2176), CSIC Researcher and Technical Director of the CSIC/UAB Proteomics Laboratory who is a proteomics expert with experience in the analysis of circulating mammalina exosomes. She has been  collaborating with the PI during the last 5 years, and will assist in Task [2.3.]\n",
        "\n",
        "### 3.3.2. Task distribution and responsibilities of the project members\n",
        "\n",
        "\n",
        "|Objective|Member       |Task|\n",
        "|---------|-------------|----|\n",
        "|1        |JC\\* (IRTA)  |Coordination of all tasks, experimental designs, sperm collection, DNA extraction|\n",
        "|         |JVR (RAB)    |Experimental designs, bioinformatic analysis of methylomics|\n",
        "|\t\t  |IF (LMU)     |Experimental designs, LC-MS and data analysis|\n",
        "|\t\t  |AR (IRTA)    |Sperm collection, semiquantitative immunoblots |\n",
        "|\t\t  |OC (IBB UAB) |Bioinformatic analysis of Hi-C|\n",
        "|\t\t  |Predoc (IRTA)|DNA extraction, LC-MS and data analysis, bioinformatic analyses of methylomics and Hi-C|\n",
        "|2        |RNF\\* (IRTA) |Coordination of all tasks, experimental designs, bioinformatic analysis of RNA-seq|\n",
        "|\t\t  |JC (IRTA)    |Experimental designs, sperm and exosome collection, RNA and DNA purification, proteomics of exosomes, primer design, FISH and IF|\n",
        "|\t\t  |JVR (RAB)    |Analysis of ChIP-seq data|\n",
        "|\t\t  |Predoc (IRTA)|Bioinformatic analysis of RNA-seq, ChIP-seq (IP and data analysis), qPCR, proteomics and immunoblots of exosomes, FISH and IF|\n",
        "|\t\t  |Tech (IRTA)  |Sperm and exosome collection, RNA and DNA purification|\n",
        "|3        |JC\\* (IRTA)  |Coordination of all tasks, experimental designs, RNA extraction, primer and morpholino design, *in vitro* sperm incubations, gene knockdown|\n",
        "|\t\t  |RNF (IRTA)   |Bioinformatic analysis of Bru-seq and embryo RNA-seq|\n",
        "|\t\t  |AR (IRTA)    |Pharmacological experiments and CASA, immunoblots, gene knockdown|\n",
        "|\t\t  |CL (INRAE)   |Experimenatl designs, pyrosequencing targeted methylation analysis|\n",
        "|\t\t  |Predoc (IRTA)|Sperm collection, *in vitro* sperm incubations and fertilization, Bru-seq and Bru-qPCR, pyrosequencing targeted methylation analysis, bioinformatic analysis of Bru-seq and embryo RNA-seq|\n",
        "|         |Tech (IRTA)  |Maintenance and breeding of zebrafish, sperm collection, RNA extraction, *in vitro* fertilization and phenotype analysis|\n",
        "\n",
        "\n",
        "- \\* = Member of the research team responsible for all tasks of the objective;\n",
        "- Predoc = predoctoral researcher to be hired;\n",
        "- Tech = laboratory technician to be hired.\n",
        "\n",
        "### 3.3.3. Chronogram of activities\n",
        "- Since BIOSPERM involves different omics technologies, and a high work load, **we propose that the  project last 4 years in order to complete all the tasks planned**.\n",
        "\n",
        "![Chronogram](https://drive.google.com/uc?export=view&id=1d-zuu_rAcZ-27w0LKJK0IK4n0xkqiTKT)\n",
        "\n",
        "- M1 (month 12): Chromatin methylation map during sperm maturation.\n",
        "- M2 (month 15): Identification of histone PTMs during sperm maturation.\n",
        "- M3 (month 24): Determination of changes in genome architecture, transcriptomics and ChIP-seq during sperm maturation.\n",
        "- M4 (month 36): RNA and protein profiling of seminal exosomes, characterization of the GnRH and PDGF signaling pathways in immature spermatozoa, and determination of hormone-induced transcription/translation mechanisms in spermatozoa.\n",
        "- M5 (month 45): Identification of endocrine regulated molecular mechanisms influencing sperm motility and inherited by the offspring.\n",
        "\n",
        "## 3.4. Identification of critical points and contingency plan\n",
        "- Although the present project is ambitious, we believe that the pilot data provided, as well as the careful  definition of the objectives, methodology and milestones in the proposal, make the project feasible  within 4 years. Moreover, although many of the tasks are complementary to bring a comprehensive view of chromatin reprogramming during the maturation of seabream spermatozoa, they are independent,  since the partial or complete failure of one of them will not compromise the success of the others.  Nevertheless, we may face some technical risks for which different **counteractive solutions** are  proposed:\n",
        "\n",
        "1. In Task [1.2.], there is a risk that no significant changes in PTMs of histones are detected between seabream immature and mature sperm. If this is the case, we will directly employ ChIP Grade  commercial antibodies against conserved histone modifications that are markers of open or close  chromatin for further ChIP-seq experiments (Task [2.2.]). The cross-reaction of some of these antibodies (H3K9me3, H3K14ac, H3K27me3, H3k4me3 and H3K79me2) with the seabream orthologs has already  been confirmed by the PI's group (Castro-Arnau et al., in preparation).\n",
        "1. Preliminary inspection of the available seabream genome assembly suggests that may be suitable  to detect genome-wide chromatin interactions during sperm maturation (Task 1.3). However, although  our Hi-C sequencing will improve the genome information for the seabream, we may encounter that the  data are finally not sufficient to construct a reliable 3D architecture of the genome. In this case, we will  employ the zebrafish model for which advanced low-cell in situ Hi-C in sperm has been carried out <sup>35</sup>. These data will be combined with those from methylomics, ChIP-seq and RNA-seq from seabream  spermatozoa to help the identification of activated or repressed chromatin regions in this species.\n",
        "Alternatively, we will employ ATAC (Assay for Transposase-Accessible Chromatin)-seq to get  information about chromatin accessibility across the seabream spermatozoon genome  <sup>36</sup>.\n",
        "3. A final risk might be that the Bru-seq experiments on seabream and zebrafish immature  spermatozoa fail due to the low input of labelled RNA after immunoprecipitation (Task 3.2). To  counteract this obstacle, we will use the Click-iTÂ® Nascent RNA Capture Kit (Invitrogen). If problems  persist, we will carry out targeted qPCR instead of library construction using primers for upregulated  transcripts coding for motility factors or other relevant RNAs identified in Tasks [2.1.] and [2.2.]\n",
        "\n",
        "## 3.5. Previous results of the research team in the topic of the proposal\n",
        "- During the last 10 years the PI's group has carried out pioneering studies on the molecular mechanisms  involved in the regulation of sperm motion kinetics in the seabream, and particularly on the role of  multiple aquaporins in activated and post-activated spermatozoa. In addition, in collaboration with the  company Rara Avis Biotec, the group has led the development of recombinant gonadotropin-based  hormone therapies to increase sperm production and quality in oligospermic flatfishes of commercial  interest, such as the Senegalese sole (*Solea senegalensis*).\n",
        "- Most recently, the PI's group described for the first time in fish the transcriptomic dynamics during  seabream spermiogenesis, which revealed the expression of multiple ion channels in ejaculated  spermatozoa. These findings uncovered the role of some of these channels, as well as of their interactions  with aquaporins, in the control of the post-activated spermatozoon swimming performance <sup>37</sup> and volume  regulatory mechanisms (Castro-Arnau et al., in preparation). More importantly, however, this study also  provided strong evidence for the potential role of the GnRHR and PDGF signaling pathways during the  differentiation and maturation of spermatozoa <sup>19</sup>. In addition, proteomics and functional analyses also  carried out by the PI's group strongly suggest that the seabream immature spermatozoon is  translationally competent (Castro-Arnau et al., in preparation). Based on these findings, the group has  gained pilot data demonstrating the existence of GnRH and PDGF signaling pathways in the fish ETDs,  which regulate sperm motility and possibly activate epigenetic factors and other sperm factors that are  transmitted to the offspring.\n",
        "\n",
        "## 3.6. Human, material and equipment resources available for the execution of the project\n",
        "- The project will be carried out at the IBB-UAB (https://ibb.uab.cat/), where the CMP group is located since 2019 (under an agreement IRTA-UAB) and has established a network of collaborators.\n",
        "- The Applicant group has all the necessary equipment (microinjection system, nucleic and protein  electrophoresis, western blot equipment, thermocyclers, temperature-controlled incubators, osmometer,  gel imaging systems, CASA), except a computer with high RAM memory for some of the bioinformatic  analyses (requested in the budget, see Section [5]), to successfully carry out the project. In addition, the  group has produced a new integrative gilthead seabream genome annotation (https://denovo.cnag.cat/Saurata), and a collection of seabream-specific antibodies against different sperm proteins.\n",
        "- The IBB-UAB is fully equipped to conduct studies on cellular and molecular biology  (ultracentrifuges, high speed centrifuges, autoclaves, milliQ water, lyophilizer, bacterial shakers, plate  readers, HPLC, cold rooms, radioactivity lab), and has also in-house research facilities with technical  managers, such as flow cytometry and FACS, cell culture, genomics, and a protein production platform.\n",
        "- The PI will also have access to UAB's Scientific and Technical Services (https://www.uab.cat/web/research/scientific-technical-services/scientific-and-technical-services--1345667278676.html),  such as luminescence lab (qPCR, plate readers), histology labs, imaging microscopy (fluorescence  microscopy and high-resolution confocal microscopy), electron microscopy, and the AQUAB facility  for zebrafish husbandry.\n",
        "- For reasons of proximity to the PI's lab, the seabrem will be maintained at the Zona d'Aquaris  Experimentals (ZAE) of the Institute of Marine Sciences (ICM-CSIC) (https://www.csic.es/es/investigacion/catalogo-de-servicios-cientifico-tecnico/unidades-de-servicio/zona-de-acuarios-y) in Barcelona. Illumina DNA sequencing will be carried out at CNAG and CRG  (https://www.cnag.crg.eu/; Barcelona), and in the commercial company Macrogen (https://dna.macrogen.com/#). LC-MS will be carried out at the Proteomics Facility from the Biomedical Center of  LMU (https://www.proteinanalytik.abi.med.uni-muenchen.de/index.html; Germany), and shotgun LC MS/MS will be conducted at the CSIC/UAB Proteomics Laboratory, IIBB-CSIC (https://www.iibb.csic.es/en/services/proteomics). Nanoparticle (exosome) fluorescent tracking analysis will be performed using the NanoSight  NS300 at the Unique Scientific and Technical Infrastructures (ICTS) NANBIOSIS (https://www.nanbiosis.es/) housed at the Institute of Materials Science of Barcelona (ICMAB-CSIC) located in the  UAB campus. The pyrosequencer PyroMark Q96 is available at LPGP-INRAE.\n",
        "\n",
        "#4 EXPECTED IMPACT OF THE RESULTS\n",
        "\n",
        "## 4.1. Expected impact on the generation of scientific-technical knowledge in the thematic  area of the proposal\n",
        "- The paternal intergenerational effects related to epigenetic modifications, including the transfer of  ncRNAs, are of increasing interest in life science research. However, the mechanisms of paternal  epigenetic marks and ncRNAs inheritance at fertilization and during embryo and offspring viability are  still a matter of study. Information on these mechanisms in piscine model species, such as the zebrafish  and salmonids, is expanding, while they remain completely unknown in most marine fish of commercial  interest. In addition, most studies on fish have focused on the influence of the parental conditions on the  epigenetic marks of ejaculated spermatozoa, rather than on the developmental regulation of  transcriptional and epigenetic processes in the spermatozoa. The BIOSPERM project aims to investigate for the first time the reprogramming of the epigenome during the maturation of seabream spermatozoa,  and how endocrine signaling pathways and the delivery of vital factors via soma to germ cells exosomes  can regulate this process and conform the final molecular cargo of the mature spermatozoon. Therefore,  the research proposed is **highly innovative and will have a high impact in the generation of entirely new knowledge in the field**, with both scientific and technical implications. The expected impacts of  the results of the project are summarized below:\n",
        "    1. The results will provide a comprehensive picture of how the chromatin and the epigenome are  reprogrammed during the maturation of fish spermatozoa. These findings will open new avenues for  understanding the impact of nutritional and culture conditions, as well as of stressors and pollutants, on  the paternal epigenetic information inherited by the offspring. In addition, they will provide the basis to  develop molecular markers of sperm quality, which will have a broad impact on fish reproduction research and aquaculture.\n",
        "    2. The elucidation of the intracellular transduction events regulated in the spermatozoa by the recently discovered endocrine signaling pathways in the fish ETDs, will uncover novel molecular mechanisms involved in the acquisition of the spermatozoon fertilization potential not previously  described in vertebrates. In addition, these studies may reveal for the first time that the transcriptional  quiescence of post-meiotic spermatozoa is not a pan vertebrate phenomenon.\n",
        "    3. The characterization of the molecular composition of seminal exosomes and its contribution to  the sperm molecular cargo will provide additional non-nuclear RNA and protein markers of sperm  quality, which could be detected in the male seminal fluid for the monitorization of the fertilization  success. In addition, knowledge of the molecular composition of the exosome membranes, as well as of  the specific membrane proteins that promote their interaction with the spermatozoa (ligands and  receptors), may allow the future design of nanoparticles (liposomes) for drug or RNA delivery with therapeutic purposes or to investigate the role of paternal RNAs on sperm function and early  embryogenesis using RNAi methods.\n",
        "\n",
        "## 4.2. Social and economic impact of the expected results\n",
        "- Aquaculture currently contributes ~40% of the seafood consumption worldwide but the future  sustainability of this industry faces important challenges. Among these, there is an increasing concern  on how the large array of environmental constraints to which cultured fish populations are submitted,  such as nutrition, hormonal treatments, xenobiotic exposure or cryopreservation, can influence the intergenerational viability of the progeny through epigenetic alterations in spermatozoa, some of which  not being readily detectable at first glance. Therefore, to assure a commercially and ecologically  sustainable aquaculture there is an urgent need for paternal epigenetic and molecular biomarkers that can incorporate information regarding the effects of the environment and culture conditions on fish  health and disease, and monitor the effect of applied therapies. The present project is aimed to this  objective by the generation of novel knowledge for the development of physiological and epigenetic  biomarkers of sperm quality in the gilthead seabream, one of the most important species in the  Mediterranean aquaculture. These markers will be very useful as diagnostic tools to evaluate the  fertilization success and early embryo development for broodstock selection in selective breeding, and  for the optimization of methods for spawning induction, *in vitro* fertilization and sperm  cryopreservation. The availability of these biomarkers may contribute to reduce the economic costs for  a sustained production of high quality fish, while maintaining an environmentally friendly aquaculture.  Therefore, the results of this project will provide an added value for the Spanish and European  aquaculture industry resulting in economic and social benefits in the medium and long-term.\n",
        "\n",
        "\n",
        "## 4.3. Plan for scientific communication and internationalization of the results (indicate  the prevision of open access publications)\n",
        "The plan to disseminate the results to the national/international scientific community under an \"open  science\" criteria, attending the recommended/mandatory practices in Horizon Europe, is as follows:\n",
        "\n",
        "1. **Publication of results**. The results of the project will be published in open access (OA) international scientific journals under Creative Commons open-copyright licences CC-BY or CC-0 to  enable the **immediate free distribution**. **A total of 5 OA publications are expected to be generated from the project** in the areas of reproductive biology, epigenetics, developmental biology and  aquaculture. We will target OA-only journals with the highest impact possible, as well as hybrid OA  journals from editorials having a 'Read-and-Publish OA' deal with IRTA and UAB (Springer, Elsevier  and Wiley), which will allow us free OA publishing. Exceptionally, to publish oustanding results of  wide interest, no OA journals, or journals from editorials for which we do not have agreements, may be selected. In this case, we will make sure that no embargo policies are applied, and that in any case the accepted manuscripts (preprints) can be immediately deposited in public repositories from IRTA  (http://repositori.irta.cat), and UAB (Digital Document Repository, DDD; https://ddd.uab.cat). These  are trusted digital archives that meet the FAIR (Findable, Accessible, Interoperable, and Reusable)  principle, and that manage, disseminate and preserve the scientific, educational and institutional  production from both institutions. The full PDFs from all the other publications, as well as the PhD  thesis expected to be produced in the project (see Section [6]), will also be placed in the same repositories.\n",
        "2. **Free access to research data**. All the data generated in the project will be deposited and stored  in different public repositories as described below in Section [4.6.] (data management plan).\n",
        "3. **Dissemination of results in national and international scientific meetings**: The results will be  presented in meetings and workshops in the fields of aquaculture, reproductive and developmental  biology, comparative physiology and genomics. **The attendance to 4 international and 3 national  meeting is planned**. The results of the project will also be presented and discussed in postgraduate and  PhD courses, such as the Master in Aquaculture from the Universidad de Barcelona, UAB, and  Universidad PolitÃ©cnica de CataluÃ±a, in which the PI teaches every year.\n",
        "\n",
        "\n",
        "## 4.4. Plan for disseminating the results to stakeholders and society\n",
        "- Beyond the scientific community, we envisage two major stakeholders with interest in the project, to which we will ensure the dissemination of the results and explore any exploitable outcome:\n",
        "    - **Aquaculture industry**. As indicated above, the PI will attend one national and one international  scientific meeting on aquaculture (Spanish National Congress in Aquaculture and Aquaculture Europe,  respectively) to which representatives of the aquaculture sector also visit, and will thus allow the direct  dissemination of the project results. Additional diffusion of the project will be carried out through  technological platforms related to marine biology and aquaculture, such as Blue-net (https://www.b-blue-networkmatch.eu/) and Marina (https://www.marina-platform.eu/login), as well as by publication  of a technical paper in a widely distributed specialized magazine in the aquaculture sector, such as  HATCHERY Feed & Management, Aquaculture Magazine or Fish Farmer, as the PI has done before (https://issuu.com/aquafeed.com/docs/hatchery_feed_and_management_vol_9_issue_1/s/11868985).\n",
        "    - **Biotech industry**. Companies involved in the development of biotech applications for assisted reproduction technologies in aquaculture may also be interested in the project. Therefore, we will attend  relevant innovation events (industry congresses or biotech trade shows) within the main fields of  application to disseminate the results of the project. Among these events, is the international conference on biotechnology BIOSPAIN (https://www.asebio.com/actualidad/agenda/biospain-2023) or the 2024  conference \"Biotechnology for the Grand Challenges of our Society\" (https://www.ecb2024.com/).  \n",
        "\n",
        "\n",
        "- The dissemination of the project results to the society will be carried out through both IRTA and  UAB organizations to reach the highest impact possible. The specific actions that will be taken are:\n",
        "    - **From IRTA**, the project objectives and results will be advertised in the webs of the FundaciÃ³n  Observatorio EspaÃ±ol de Acuicultura (FOESA; http://www.fundacionoesa.es), the Catalonian Maritime  Network of R+D+I (http://www.xraq.cat/), and the aquaculture website mispeces.com (http://www.mispeces.com). In addition, IRTA's Communication Department will take the following additional  actions:\n",
        "        - (i) At least two press releases (at the launch and end of the project) will be sent to specialized  and mainstream Spanish media;\n",
        "        - (ii) General information on the project (project sheets) will be published  in English, Catalan and Spanish and posted at IRTA's website (www.irta.cat); and\n",
        "        - (iii) Project updates  and results will also be disseminated via IRTA's social media accounts, Twitter (>12,000 followers),  Facebook (>3,300 followers), LinkedIn (close to 30,000 followers) and YouTube (>3,500 subscribers,  with 12,000 views/month), by external monthly newsletters (CAT & ES)  (close to  3,000 subscribers),  as well as by internal weekly newsletters (900  followers).\n",
        "\n",
        "    - **From the UAB**, the project and results will also be disseminated through the main social media  institutional channels: Twitter (@IBB_UAB), and Instagram (ibbuab), LinkedIn (IBB UAB) and UAB's Facebook. We will also create and share videos on these social media describing interesting aspects of  fish reproduction, as well as reinforcing the importance of Spanish and European research in  aquaculture. The results of the project will also be communicated by writing pieces for non-specialists  in newsletters, press releases through the UAB's Communication Department (e.g. see example at  https://www.uab.cat/web/sala-de-premsa/detall-noticia/un-nou-mecanisme-podria-explicar-com-els-espermatozoides-de-peixos-poden-fertilitzar-en-el-medi-aquatic-1345829508832.html?detid=1345838443320), as well as through the UAB magazine Divulga (http://www.uab.cat/uabdivulga). The  UAB website has 4 million visitors/year, and total followers of UAB's social networks are ~150.000.  The impact of the project related actions will be assessed by the number of visitors of IBB and group  website, as well as by the number of video watches.\n",
        "\n",
        "\n",
        "## 4.5. Plan for the transfer and valorization of the results\n",
        "\n",
        "1. **Transfer of knowledge to the industry**. Since 2017, the PI is collaborating with the company Rara Avis Biotec, which is specialized in the production and use of recombinant gonadotropins to induce  reproductive cycles in vertebrates, and with which two joint transfer technology projects have recently  been carried out (CDTi BIOTECLEN and EU ESIF REARLING). The PI also has contacts with the  company IMV Technologies (France) (https://www.imv-technologies.com/), leader in the areas of  animal and human assisted reproduction technologies. Both companies will benefit from the use of biomarkers of sperm quality to validate their technologies, and therefore they are very interested in the  outcome of the BIOSPERM project. Indeed, one member of Rara Avis Biotec will participate in the  working team of the project. In addition, the PI has also recently collaborated with the company Stolt  Sea Farm within the framework of the BIOTECLEN project, which is very interested in methods to  enhance sperm quality in oligospermic fish, such as the Senegalese sole, to improve *in vitro* fertilization procedures. Our preliminary data show that hormones can increment the motility of the sole spermatozoa  as in seabream, and therefore these methods will be of high interest to this company. Three separate  formal meetings with the three companies will be organized during the last two years of the project in  which we will share and discuss the results and seek for joint technology transfer actions.\n",
        "2. **Patenting**. In biomedical research, the development of methods for the monitorization of biomarkers for cancer diagnosis has recently become of high interest and can be patented. A similar  approach will be explored if useful sperm quality biomarkers are identified in the project. To reach this, the PI will have the support of experts on innovation and technology transfer from IRTA's Innovation  & Business Development Office, who will assist in the study of the protection/patentability options of  the invention(s), and if decided, in the drafting of a patent application. For this, we will also consider  applying for additional external funding from the Catalan Government (GÃ­njol Patents Fund, Llavor Grants) and the Spanish Research Agency (AEI, Proof of Concept projects).\n",
        "\n",
        "## 4.6. Summary of the planned data management plan\n",
        "- The PI will be responsible that all data are deposited in institutional national and/or international trusted  repositories that meet the FAIR principle during the course of the project, and in any case within two  years after its completion.\n",
        "- The **genomic and proteomic data** will be deposited and stored in different public repositories. Thus,  all raw and processed sequencing data generated will be submitted to the Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) database at the National Center for Biotechnology  Information (NCBI), whereas the MS proteomic data will be deposited in the ProteomeXchange (PX)  consortium (http://proteomecentral.proteomexchange.org). The chromatin interaction maps generated  by Hi-C and raw data will be submitted to the 4D Nucleome data portal (https://data.4dnucleome.org/)\n",
        "- All **other final data** (numerical data used to generate published graphs and tables, photomicrographs,  gels and immunoblots) will be deposited in the CORA Research Data Repository (CORA.RDR)  (https://dataverse.csuc.cat/).\n",
        "\n",
        "## 4.7. Effects of the inclusion of gender in the content of the proposal.\n",
        "- **Research focus**: Because of the objectives of the proposed research are focused on spermatozoa, the  project is necessarily biased to the fish male sex/gender.\n",
        "- **Researchers**: The project teams include two women (25%). However, the gender dimension of the  researchers involved in the project is more equilibrated with the participation of three outstanding  women senior researchers as external collaborators (see Section [3.3.1]). To further increase the  participation of women in the project, the PI will encourage women applicants for the requested  predoctoral contract, and will favor the hiring of women in case of equally skilled men and women\n",
        "candidates.\n",
        "- **Gender equality plan**: IRTA has an Equal Opportunity Plan (https://www.irta.cat/wp-content/uploads/  2022/03/Plan-de-Igualdad-de-Oportunidades-IRTA-2022-2025.pdf) that will assure gender equality in  all aspects of the research project according to Horizon Europe Work Program for 2021-22. The UAB  has also the IV Action Plan for Gender Equality (https://www.uab.cat/doc/IVplageneredoc), which has  been prepared by the Observatory for Equality at UAB. The plan articulates policies for gender equality  and LGTBI collective (lesbians, gays, bisexuals, transsexuals and intersex) at the university level.\n",
        "- **Communication**: The project results will be communicated using a non-sexist language.\n",
        "\n",
        "#5 JUSTIFICATION OF THE REQUESTED BUDGET  \n",
        "- The budget **rationalization** is indicated below (quantitative justifications are available in the application  document):  \n",
        "\n",
        "    - **Personnel (Laboratory Technician)**: The applicant group does not have a technician who can help in the high time-consuming tasks included in the project, such as the collection of seabream sperm and  exosome samples, the maintenance of a continuous supply of zebrafish mature males and embryos, the  conduction of *in vitro* fertilization trials and embryo phenotype analysis, and the extraction of sperm  nucleic acids. Therefore, a 2-year full-time contract for an initial technician trainee (IRTA level 1.1), with professional training qualification (FP) as Higher Technician in Aquaculture or in Clinical and  Biomedical Laboratory, is requested from month 18 to 42 to assist in these tasks (defined in Section [3.3.2.]). The position will also offer an excellent opportunity for training in seabream culture and the use of piscine model systems for biological research, and it would be very beneficial to optimize the time  that the researchers should spend on the project.\n",
        "    - **Equipment**: A computer (CELSIUS M7010POWER) is requested to process NGS data from ChIP-seq  and Hi-C by the PI's group, since the bioinformatic analyses associated to these specific methods have to be carried out by the research group and not by the sequencing services. The algorithms and  bioinformatic tools used in ChIP-seq and Hi-C (see Section [3.2.]) are known to be computationally  intensive and consume very high RAM memory. For this reason, it is necessary to acquire a computer  with much more RAM memory than the one found in common computers. The PI's group does not have  such a computer, and therefore one with 512 Gb of RAM, an amount that is considered sufficient for  the processes that will be carried out in the project, is solicited.\n",
        "    - **Consumables**: Costs of general and specific laboratory consumables necessary for all tasks of the  project, including molecular biology kits and reagents, antibodies, microinjection material, hormones,  pyrosequencing and fluorometric kits. Note that some kits and commercial antibodies (e.g. ChIP grade  antibodies against modified histones) are very expensive (500-1000â¬). Here, is also enclosed the costs  of the purchase, transport and maintenance of seabream (3 years) and zebrafish (2.5 years) at the ICM CSIC and AQUAB facilities, respectively.\n",
        "    - **Other costs**: Costs of Illumina sequencing services at CNAG, CRG and Macrogen, and of proteomics  at LMU and the Lab. CSIC/UAB, which are essential for the project. The use of different sequencing platforms, sometimes at the same time, will optimize the time to complete the multiple sequencing tasks  throughout the project, or comes from the specific expertise of one of the platforms (RNA-seq of  sncRNAs at the CRG). The charges for the use of FACSJazz, electron and confocal microscopy services  at UAB, of NanoSight NS300 at NANBIOSIS, as well as the costs for OA publishing, are also included.  Note that publishing in OA-only journals requires unavoidable payment (2-3 publications, out of 5, are  expected in these journals). The costs for the maintenance and repair of equipments intensively used  during the project (centrifuges, freezers and termocyclers), as well as for studies of the protection and  patentability options of inventions (including state-of-the art searches), are also enclosed.\n",
        "    - **Travel**: Attendance to 4 international and 3 national scientific conferences or workshops of two project  members (from the research and working teams), as well as PI's attendance to one national and one  international biotech trade shows, to disseminate the results of the project. Includes also one PI's visit  of 2 days to the proteomics LMU lab (Germany) for a follow-up meeting and discussion of results, and  two visits of 2 days each of the PI to the companies IMV Technologies (France) and Rara Avis Biotec (Valencia) for the sharing of results and discussion of joint technology transfer activities.\n",
        "- The mean amount of funding for consumables and travel requested is of ~3.600â¬/researcher/year,  which we believe is very adjusted if one considers the number of different tasks that are going to be  carried out in the project and its extension of 4 years.\n",
        "\n",
        "#6 TRAINING COMPETENCE\n",
        "\n",
        "## 6.1. Interest of the proposal for the training of predoctoral researchers and hiring  process  \n",
        "- The project offers a unique opportunity to carry out a PhD thesis on fish molecular reproductive biology because the research planned is novel, carried out by a highly collaborative interdisciplinary team, and  of high scientific and applied interest. The project will offer to the predoctoral researcher a deep training  in cell and molecular biology, state-of-the art genomics technologies and bioinformatic analyses, biochemistry and spermatology, with an applied orientation towards biotechnology in aquaculture.\n",
        "- The predoctoral researcher will be hired through IRTA, which holds the EURAXESS distinction  \"HR Excellence in Research\". This award gives public recognition to research institutions that have  made progress in aligning their human resource policies with the principles set out in the \"European  Charter for Researchers\" and the Code of Conduct for the Recruitment of Researchers. Therefore, IRTA is committed to implement fair and transparent recruitment and appraisal procedures for researchers.\n",
        "\n",
        "## 6.2. Training program planned in the context of the requested project\n",
        "- The PhD researcher will be integrated in one of the UAB PhD programs relevant to the topic of the  thesis, such as Biotechnology, Cell Biology, Biochemistry, Molecular Biology and Biomedicine or Bioinformatics (because of the high component in computational biology included in the thesis project).  The thesis will adjust to the rules of the UAB to have the **mention of European PhD**.\n",
        "- In addition to the courses included in the PhD program, a training plan for the PhD researcher will  be designed with the objective that she/he increments his/her scientific and communication skills  through experience and knowledge, as well as her/his personal maturity and professional network. The  training plan is outlined below:\n",
        "1. **Technical training**. The researcher will be trained in two different areas:\n",
        "    - (a) <ins>Laboratory techniques</ins>: She/he will be trained in molecular biology methods currently employed  by the PI's group, such as cloning, qPCR, immunopreciptation and immunoblotting, histology,  immunofluorescence microscopy, exosome purification and characterization, as well as in CASA, in  vitro culture of sperm and fertilization, and embryo phenotyping. This training will be acquired by the  PhD researcher while she/he is working in specific tasks of the project defined in Section [3.3.2.]\n",
        "    - (b) <ins>Bioinformatics</ins>: The PhD researcher will also be trained in different bioinformatic methods and  tools to analyze RNA-seq data (gene ontology classification, functional annotation of differentially  expressed genes/proteins, pathways analysis), as well as methylomics, ChIP-seq and Hi-C data  (identification and quantification of DMRs, read distribution and chromatin state annotation and  clustering, chromatin compartmentalization, analysis of topologically associated domains).\n",
        "2. **Short training visits in external laboratories**. Further specific training of the predoctoral  researcher will be carried out through short visits to external laboratories as follows:\n",
        "\n",
        "| Laboratory                        | Advisor         | Duration (months) | Specific training                             |\n",
        "| ----------------------------------| ----------------| ----------------- | --------------------------------------------- |\n",
        "| LMU, Germany                      | Dr. I. FornÃ©    | 3                 | Analysis of histone PTMs by high-resolution MS|\n",
        "| Lab Proteomics CSIC/UAB, Barcelona| Dr. M. Carrascal| 3                 | Label-free LC-MS/MS shotgun proteomics        |\n",
        "| CRG, Barcelona                    | Dr. F. Le Dily  | 3-5               | Analysis of Hi-C data                         |\n",
        "| LPGP-INRAE, France                | Dr. C. LabbÃ©    | 3                 | Pyrosequencing targeted methylation method    |\n",
        "\n",
        "3. **Attendance to scientific seminars and specific courses**. The PhD researcher will attend and  actively participate in the IBB Seminar's program (available at the IBB webpage), which is active all year round and includes invited national and international speakers. Comprehensive training will also  be provided by the attendance to different courses offered by IRTA and UAB, such as \"*Curso oficial  homologado de experimentaciÃ³n de Animales de Laboratorio* (UAB)\" (mandatory for all IBB PhDs using animal models), \"*Presenting Research*\" or \"*Scientific writing*\". The researcher will also be  allowed to attend for other external courses (i.e. computational biology), that can improve her/his skills.\n",
        "4. **Mentoring**. In periodic laboratory meetings of the PI's group, the PhD researcher will present  their own results and discuss papers from other laboratories working in the same or related fields. The  PI will have biweekly individual meetings with the researcher to guide the experimental work, discuss  the results, and organize her/his attendance to scientific meetings and short training visits.\n",
        "\n",
        "## 6.3. PhD thesis carried out or in progress in the research team (last 10 years)\n",
        "- The PI has advised 6 PhDs (plus 3 more in progress) and 4 Master thesis (2011, 2013, 2018, and 2019),  which demonstrates his experience in the training of graduate and predoctoral students. **Two of the PhD  thesis supervised obtained the PhD Extraordinary Award of University of Barcelona (2005 and  2013)**. The thesis advised by the PI during the last 10 years are listed below:\n",
        "1. [**\"*Molecular physiology of a teleost oocyte aquaporin: evolution, regulation, and role during oocyte hydration*\"**](https://digital.csic.es/handle/10261/130835?mode=full). Cinta Zapater Cardona, 2013. University of Barcelona. Suitable \"*cum laude*\".  **EXTRAORDINARY PhD AWARD OF UNIVERSITY OF BARCELONA**. European PhD. <ins>Publications</ins>: 10 SCI articles.\n",
        "2. [**\"*Aquaporin biology during spermatogenesis and sperm physiology in the marine teleost gilthead  seabream (Sparus aurata)*\"**](https://rua.ua.es/dspace/handle/10045/57628#vpreview). MÃ³nica Boj LidÃ³n, 2016. Co-advised with FranÃ§ois ChauvignÃ© (CSIC).  University of Alicante. Excellent \"*cum laude*\". European PhD. <ins>Publications</ins>: 6 SCI articles.\n",
        "3. **\"*Functional evolution of aquaporin trafficking in fish oocytes*\"**. Alba FerrÃ© GarcÃ­a. Co-advised  with FranÃ§ois ChauvignÃ© (CSIC). University AutÃ³noma of Barcelona. <ins>Publications</ins>: 3 SCI articles.  To be defended in 2023.\n",
        "4. **\"*Molecular regulation of fish spermiogenesis and role of ion channels*\"**. Julia Castro i Arnau.  University Pompeu Fabra. <ins>Publications</ins>: 5 SCI articles. To be defended in 2023.\n",
        "5. **\"*Aquaporin-based biotechnologies for the cryopreservation of fish oocytes and embryos*\"**. Angel  Rey Alfonso. University AutÃ³noma of Barcelona. In the second year.\n",
        "\n",
        "## 6.4. Scientific or professional development of the graduated doctors\n",
        "- PhD students have taken careers in science either in the academic or industry sector.  The current jobs of the PI's former students are listed below:\n",
        "\n",
        "| Name                 | Year of Thesis defense | Current position|\n",
        "| ---------------------| ---------------------- | --------------- |\n",
        "| Mercedes Fabra       |2005                    | Project Leader, Immunology and Vaccines for Global Health, Laboratorios LETI S.L., Spain (â  January 15, 2020).|\n",
        "| MarÃ­a del Mar Huertas|2006                    | Assistant Professor, Texas State University, USA.|\n",
        "| MarÃ­a J. Agulleiro   |2007                    | School Professor, Departament d'Ensenyament, Generalitat de Catalunya, CastellÃ³n, Spain.|\n",
        "| Ignasi FornÃ©         |2007                    | Technical Director, Protein Analysis Unit, Ludwig Maximilian University of Munich, Germany.|\n",
        "| Cinta Zapater        |2013                    | Contracted Researcher, IATS-CSIC, Spain.|\n",
        "| MÃ³nica Boj           |2016                    | Non-scientific related.|\n",
        "\n",
        "# 8 SPECIFIC CONDITIONS FOR THE EXECUTION OF CERTAIN PROJECTS\n",
        "\n",
        "- **Experimental animals and facilities**: The seabream will be obtained from commercial farms and/or national public aquaculture research stations and maintained at the ZAE (ICM-CSIC) (http://www.cmima.csic.es/serveis/zae/index.php) (Registration No. 08039-46-A). Wild-type zebrafish AB strain  will be obtained from the European Zebrafish Resource Center (https://www.ezrc.kit.edu/) and maintained at the AQUAB fish facilities (Registration No. B9900080).\n",
        "- **Licenses**: The project PI is entitled as Competent Personnel to work on animal experimentation given  by the Government of Catalonia. Another member of the working team (A. Rey) has completed an  official training course to wok on animal experimentation given by the UAB, which allows him to use  animal models. The predoctoral researcher to be hired will take the same course.\n",
        "- **Procedures**: The experimental procedures will only include the sacrifice of the animals, which can not  be considered 'animal experimentation' (see General Directorate of Environmental Policies,  Government of Catalonia). The method for sacrifice will follow the recommendations of the Directive  2010/63/EU of the European Parliament (Annex IV) and is approved by IRTA's Ethics and Animal  Experimentation Committee. Animals will be sacrificed by prolonged exposure to the anesthetic tricaine  methanesulfonate at 100 ppm by immersion, followed by the destruction of the brain by the insertion of  a needle through the foramen magnum, at the atlanto-occipital joint, to ensure the rapid and complete  destruction of the brain. Zebrafish embryos will be collected from naturally breeding fish following well  established methods (pair-wise breeding), and will be sacrificed at 5 days post-fertilization.\n",
        "- **3R Principle**: Simulation of the experiments using cell lines is not possible. Questions in this project  can only be answered by experiments with living fish. The minimum number of sacrificed animals has  been estimated to be ~70 seabream males and ~250 zebrafish males to assure statistically consistent results using the minimum number of replicates after calculation with the software G Power <sup>38</sup>.\n",
        "\n",
        "# 7 REFERENCES (publications by project members and collaborators are in bold)\n",
        "1. [Migaud et al. 2013 Rev Aquaculture 5:S194-S223](https://onlinelibrary.wiley.com/doi/full/10.1111/raq.12025).\n",
        "2. [Alavi et al. 2019 Theriogenology 136:143-165](https://www.sciencedirect.com/science/article/abs/pii/S0093691X19302663).\n",
        "3. [Alves et al. 2020 Front Cell Dev Biol 8:791](https://www.frontiersin.org/articles/10.3389/fcell.2020.00791/full).\n",
        "4. [Ord et al. 2020 Mol Ecol 29:2722-2735](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.15505).\n",
        "5. [Zhu et al. 2021 Front Genet 12:55793](https://www.frontiersin.org/articles/10.3389/fgene.2021.557934/full).\n",
        "6. [LombÃ³ M, HerrÃ¡ez MP. 2021. Int J Mol Sci 22: 2125](https://www.mdpi.com/1422-0067/22/4/2125).\n",
        "7. [HerrÃ¡ez et al. 2017. Aquaculture 472:45-59](https://www.sciencedirect.com/science/article/abs/pii/S0044848616301168).\n",
        "8. [Lahnsteiner F. 2003 Tissue Cell 35:363-73](https://www.sciencedirect.com/science/article/abs/pii/S0040816603000570).\n",
        "9. [**ChauvignÃ© et al. 2018 J Anat 233:177-192**](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6036930/).\n",
        "10. [Ciereszko et al. 2015 Theriogenology 83:1213-8](https://www.sciencedirect.com/science/article/abs/pii/S0093691X14007183?via%3Dihub).\n",
        "11. [Contreras et al. 2022 Rev Fish Biol Fisheries  32:331-355](https://link.springer.com/article/10.1007/s11160-021-09690-8).\n",
        "12. [Gur Y, Breitbart H. 2006 Genes Dev 20:411-6](https://genesdev.cshlp.org/content/20/4/411.long).\n",
        "13. [Rajamanickam et al. 2017 Cell Tissue Res  368:187-200](https://pubmed.ncbi.nlm.nih.gov/27807702/).\n",
        "14. [Torres-Flores U, HernÃ¡ndez-HernÃ¡ndez A. 2020  Front Genet 11:780](https://www.frontiersin.org/articles/10.3389/fgene.2020.00780/full).\n",
        "15. [Chen et al. 2022 Front Cell Dev Biol 10:834519](https://www.frontiersin.org/articles/10.3389/fcell.2022.834519/full).\n",
        "16. [Robles et al. 2019 Noncoding RNA Res 4:54-62](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6926202/).\n",
        "17. [Riesco et al. 2019 Sci Rep 9:11192](https://www.nature.com/articles/s41598-019-47702-7).\n",
        "18. [Zhang et al. 2019 Mar Biotechnol 21:310-319](https://link.springer.com/article/10.1007/s10126-019-09881-2).\n",
        "19. [**Castro-Arnau et al. 2022 Sci Rep 12:14162**](https://www.nature.com/articles/s41598-022-18422-2).\n",
        "20. [Vaisvila R, et al. 2021 Genome Res 31:1280- 1289](https://genome.cshlp.org/content/31/7/1280.full).\n",
        "21. [Li et al. 2022 Epigenomics 14:951-964](https://www.futuremedicine.com/doi/full/10.2217/epi-2022-0122?rfr_dat=cr_pub++0pubmed&url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org).\n",
        "22. [Moradian et al. 2014 Proteomics 14:489-97](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/pmic.201300256).\n",
        "23. [**Feller et al. 2015 Mol Cell 57:559-71**](https://www.cell.com/molecular-cell/fulltext/S1097-2765(14)00957-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276514009575%3Fshowall%3Dtrue).\n",
        "24. [Bourmaud et al. 2016 Proteomics 16:2146-59](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/pmic.201500543).\n",
        "25. [**Vara et al. 2019. Cell Rep 28:352-367**](https://www.cell.com/cell-reports/fulltext/S2211-1247(19)30803-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124719308034%3Fshowall%3Dtrue).\n",
        "26. [Papadaki et al. 2020 Mar Biotechnol (NY) 22:683- 695](https://pubmed.ncbi.nlm.nih.gov/32876760/).\n",
        "27. [Paulsen et al. 2014 Methods 67:45-54](https://www.sciencedirect.com/science/article/abs/pii/S1046202313003228?via%3Dihub).\n",
        "28. [**DepincÃ© et al. 2021 Sci Rep 11:3945**](https://www.nature.com/articles/s41598-021-83033-2).\n",
        "29. [Flynt et al. 2017 Methods Mol Biol 1565:59-78](https://pubmed.ncbi.nlm.nih.gov/28364234/).\n",
        "30. [**Pokrovsky et al. 2021 PLoS Biol 19:e3001377**](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001377&rev=1).\n",
        "31. [**Alabert et al. 2020 Cell Rep 30:1223-1234**](https://www.cell.com/cell-reports/fulltext/S2211-1247(19)31717-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124719317176%3Fshowall%3Dtrue).\n",
        "32. [**VÃ¶lker-Albert et al. 2018. Curr Protoc Protein Sci 92:e54**](https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpps.54).\n",
        "33. [**FornÃ© et al. 2009 Proteomics 9:2171-81**](https://pubmed.ncbi.nlm.nih.gov/19322785/).\n",
        "34. [**FornÃ© et al. 2011 Proteomics 11:2195-211**](https://pubmed.ncbi.nlm.nih.gov/21538881/).\n",
        "35. [Wike et al. 2021. Genome Res 31:981-994](https://genome.cshlp.org/content/31/6/981.long).\n",
        "36. [Bedi et al. 2022 Epigenetics Chromatin 15:2](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-021-00433-4).\n",
        "37. [**Castro-Arnau et al. 2022 Int J Mol Sci 23: 12113**](https://www.mdpi.com/1422-0067/23/20/12113).\n",
        "38. [Faul et al. 2007 Behav Res Methods 39:175-91](https://link.springer.com/article/10.3758/BF03193146)."
      ],
      "metadata": {
        "id": "gkjcYZPCI4Zh"
      }
    }
  ]
}